src/chrome/content/rules/Komoona.com.xml

```diff
@@ -8,32 +8,43 @@
 
 	At least some pages redirect to http.
 
+	TLS not supported on:
+		-	test.komoona.com - self-signed
 -->
 <ruleset name="Komoona.com (partial)">
 
 	<target host="komoona.com" />
-	<target host="*.komoona.com" />
-		<!--
-			Redirect to http:
-						-->
-		<!--exclusion pattern="^http://www\.komoona\.com/+($|\?)" /-->
-		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://www\.komoona\.com/+(?!favicon\.ico|new/css/|new/images/)" />
-
+	<target host="a.komoona.com" />
+	<target host="a1.komoona.com" />
+	<target host="a2.komoona.com" />
+	<target host="a3.komoona.com" />
+	<target host="a4.komoona.com" />
+	<target host="admin.komoona.com" />
+	<target host="cdn.komoona.com" />
+	<target host="cs.komoona.com" />
+	<target host="e1.komoona.com" />
+	<target host="e2.komoona.com" />
+	<target host="s.komoona.com" />
+	<target host="stat.komoona.com" />
+	<target host="w.komoona.com" />
+	<target host="www.komoona.com" />
+
+		<exclusion pattern="^http://www\.komoona\.com/(?!favicon\.ico|new/css/|new/images/)" />
+
+	<test url="http://www.komoona.com/favicon.ico" />
+	<test url="http://www.komoona.com/new/css/" />
+	<test url="http://www.komoona.com/new/images/" />
 
```

Comment:
Then you need to add 3 more test urls that match the exclusion.

FYI, only `https://www.komoona.com` redirects to http. The other pages have a font is blocked but it doesn't look broken. I am not a official reviewer, but I would suggest just exclude `^http://www\.komoona\.com/$`